stance/etl/template_fit.py

```diff
@@ -0,0 +1,72 @@
+import cv2
+
+
+def _get_points(image):
+    """
+    COCO Output Format
+    Neck – 1, Right Shoulder – 2, Right Elbow – 3, Right Wrist – 4,
+    Left Shoulder – 5, Left Elbow – 6, Left Wrist – 7, Right Hip – 8,
+    Right Knee – 9, Right Ankle – 10, Left Hip – 11, Left Knee – 12,
+    LAnkle – 13
+
+    Parameters
+    ----------
+    image - image of user
+
+    Returns
+    -------
+    points - points relating to body parts of the user from the image
+        a certain point is None if no relative point is found
+    """
+    proto_file = "../../model/pose_deploy_linevec.prototxt"
+    weights_file = "../../model/pose_iter_440000.caffemodel"
+    n_points = 18
+
+    frame = cv2.imread(image)
+    frame_width = frame.shape[1]
+    frame_height = frame.shape[0]
+    threshold = 0.1
+
+    net = cv2.dnn.readNetFromCaffe(proto_file, weights_file)
+
+    # t = time.time()
+    # input image dimensions for the network
+    in_weight = frame.shape[1]
+    in_height = frame.shape[0]
+    inp_blob = cv2.dnn.blobFromImage(frame, 1.0 / 255, (in_weight, in_height),
+                                     (0, 0, 0), swapRB=False, crop=False)
+
+    net.setInput(inp_blob)
+
+    output = net.forward()
+    # print("time taken by network : {:.3f}".format(time.time() - t))
```

Comment:
We might want to turn this into a proper `logging` thing if we end up caring about the timing